dm/pkg/utils/db.go

```diff
@@ -361,7 +361,7 @@ func GetGTIDMode(ctx context.Context, db *sql.DB) (string, error) {
 
 // ExtractTiDBVersion extract tidb's version
 // version format: "5.7.25-TiDB-v3.0.0-beta-211-g09beefbe0-dirty"
-//                               ^~~~~~~~~^
+// -                            ^..........
```

Comment:
```suggestion
// to be extracted:              ^~~~~~~~~^
```

Reply:
It's a workaround for them. I tried a lot of ways and only this way was more accurately with our intention.